### PR TITLE
Fixes #9594 - adds admin field to the hostgroup rabl

### PR DESCRIPTION
**Pull request by ShimShtein**

### CODE REVIEW

File: app/views/api/v2/usergroups/main.json.rabl [hunk 1]
```diff
@@ -2,4 +2,4 @@ object @usergroup
 
 extends "api/v2/usergroups/base"
```
Comment (domcleal) [inline app/views/api/v2/usergroups/main.json.rabl:3]: move :admin to the base file I'd say?

Comment (domcleal) [inline app/views/api/v2/usergroups/main.json.rabl:3]: Oh, never mind, it's here in users.

Comment (ShimShtein) [inline app/views/api/v2/usergroups/main.json.rabl:3]: users table has a different `:admin` field. Anyway, I'm not sure that the field is relevant to all views.

**Comment by dLobatog** [comment]
:+1: if Jenkins approves.

**Comment by dLobatog** [comment]
Merged as 57ccff8bc311d394beb911dda9e178ec445ad097, thanks @ShimShtein!

**Pull request closed without merging**